apps/designer/app/designer/shared/image-manager/image-thumbnail.tsx

```diff
@@ -98,7 +90,8 @@ export const ImageThumbnail = ({
 
   const isUploading = status === "uploading";
 
-  const src = useImageWithFallback({ path });
+  // Must be the same as the Thumbnail width "$spacing$19"
+  const imageWidth = getInt(theme.spacing[19].value) ?? 64;
```

Comment:
I think it's safe to assume `theme.spacing[19].value` is a number or use some simple check, `getInt()` seem like an overkill

Reply:
its not a number, its 64px now